packages/integrations/node/test/errors.test.js

```diff
@@ -17,10 +22,30 @@ describe('Errors', () => {
 	let devPreview;
 
 	before(async () => {
-		devPreview = await fixture.preview();
+		// devPreview = await fixture.preview();
```

Comment:
To remove

Reply:
It isnt being used yet

Reply:
What do you mean with yet? Can you be more precise, please?

Reply:
The other two tests in the suite are skipped. The one I added starts a server within a worker. The preview server being started here is not used.

Reply:
and it listens on the same port

Reply:
Usually, if you want to keep commented code around, it's customary to leave a comment explains that :)

Reply:
Sure ill add that

Reply:
```suggestion
		// The two tests that need the server to run are skipped
		// devPreview = await fixture.preview();
```